src/Model/TransportModel/tsp-apt.f90

```diff
@@ -635,6 +635,19 @@ function apt_check_valid(this, itemno) result(ierr)
     end if
   end function apt_check_valid
 
+  !> @brief Advanced package transport utility function
+  !!
+  !! Set the concentration (or temperature) to be used by either MVT or MVE
+  !<
+  function get_mvtconc(this)
```

Comment:
Do we want to call this get_mover_depvar instead (here and elsewhere in the changes) so that it works for both concentration and temperature?  Just a thought.

Reply:
Good idea @langevin-usgs, updated function name with 8ef02be